gpkit/varkey.py

```diff
@@ -24,6 +24,8 @@ class VarKey(object):
     VarKey with the given name and descr.
     """
     new_unnamed_id = Counter()
+    subscripts = ["model", "idx"]
+    eq_ignores = frozenset(["units", "value"])
```

Comment:
Embarrassing question I'll ask anyway: 

You said:  "now that unitstr are compared it's basically like units are compared", but looking at the current diff I see that unitstr was added to eq_ignores: `eq_ignores = frozenset(["units", "unitstr", "value"])`. So, is unitstr being compared, or ignored??